### ISSUE TITLE
(Un)Safe Mode Bypasses FTBU Claims

### ISSUE THREAD

**Issue by Agr10n** [pWn3d1337/Techguns2#51]
Having some griefing issues, it seems "safe mode" is a client side check and players with it off are still able to damage player claims (through FTBU) even though "explosions" are off. I verified that all other explosions not from Techguns, creeper, tnt, etc do no damage. I have dug all over the interwebs, and I see that old Techguns for 1.7 had a config option to disable "unsafe mode" except for OPs. I have tried everything I can think of including several possible permissions but nothing seems to change. I would appreciate a quick response before griefing leads to constant rollbacks, at least FTBU has server backups but obviously this is not ideal. Thanks in advance! It really is a great mod and I'd hate to remove it for one fatal flaw.

**Comment by pWn3d1337** [comment]
https://github.com/pWn3d1337/Techguns2/blob/master/src/main/java/techguns/TGPermissions.java

**Comment by Agr10n** [comment]
Thanks for the response but still not exactly clear. When using FTBU ranks system the permissions are "ftbutilities.home: true" for example, but also mostly specific commands or permissions. I have tried every variation I can think of and my default "player" rank still can switch to unsafe mode at will. I have tried:
techguns.allowunsafemode: false
techguns.modid+".allowunsafemode": false
techguns.allow_unsafe_mode: false
techguns.techguns,allowunsafemode: false
techguns2.allowunsafemode: false
allowunsafemode: false

Not sure what to do next. Main issue is ftbu specifies whole thing in quotes like (String CHAT_SPEAK = "ftbutilities.chat.speak";) where yours throws "MODID+" in the middle and quotes start at ".allowunsafemode" so I'll keep trying different ones I guess. Sorry for being a noob I can usually figure this stuff out but not having any luck this time. Still a great mod, but I have to say documentation is few and far between, JEI bridges most gaps but not all. Thank you for your time.
